diff --git a/node_modules/electron/electron.d.ts b/node_modules/electron/electron.d.ts
index 3fa42e0..93762d8 100644
--- a/node_modules/electron/electron.d.ts
+++ b/node_modules/electron/electron.d.ts
@@ -9867,23 +9867,23 @@ declare namespace Electron {
      *
      * @platform darwin,win32
      */
-    on(event: 'speed-limit-change', listener: () => void): this;
+    on(event: 'speed-limit-change', listener: (limit: number) => void): this;
     /**
      * @platform darwin,win32
      */
-    off(event: 'speed-limit-change', listener: () => void): this;
+    off(event: 'speed-limit-change', listener: (limit: number) => void): this;
     /**
      * @platform darwin,win32
      */
-    once(event: 'speed-limit-change', listener: () => void): this;
+    once(event: 'speed-limit-change', listener: (limit: number) => void): this;
     /**
      * @platform darwin,win32
      */
-    addListener(event: 'speed-limit-change', listener: () => void): this;
+    addListener(event: 'speed-limit-change', listener: (limit: number) => void): this;
     /**
      * @platform darwin,win32
      */
-    removeListener(event: 'speed-limit-change', listener: () => void): this;
+    removeListener(event: 'speed-limit-change', listener: (limit: number) => void): this;
     /**
      * Emitted when the system is suspending.
      */
